themeapp/templates/widgets/top-bar.html

```diff
@@ -10,7 +10,6 @@
                 <li class="nav-item"><a class="nav-link" href="{% url 'account_logout' %}">Log Out</a></li>
                 {% else %}
                 <li class="nav-item"><a class="nav-link" href="{% url 'account_login' %}">Log In</a></li>
-                <li class="nav-item"><a class="nav-link" href="{% url 'account_signup' %}">Sign Up</a></li>
                 {% endif %}
```

Comment:
Again. Separate PR for separate functionality. I would merge this change immediately.